package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "react-facebook-login",
-  "version": "2.2.0",
+  "version": "3.0.0",
   "description": "A Component React for Facebook Login",
   "main": "dist/facebook-login.js",
   "scripts": {
-    "start": "webpack-dev-server --config webpack.config.js",
-    "bundle": "webpack -p --config webpack.config.dist.js",
+    "start": "NODE_ENV=development webpack-dev-server --config webpack.config.js --host $IP --port $PORT",
```

Comment:
@anthonyjgrove why you set $IP and $PORT, i think this is bad, but a need set in my bash or export IP and PORT to run project.